src/Sarif.Driver/Sdk/SarifLogger.cs

```diff
@@ -32,36 +33,102 @@ public static ToolInfo CreateDefaultToolInfo(string prereleaseInfo = null)
             return toolInfo;
         }
 
+        private static RunInfo CreateRunInfo(
+            IEnumerable<string> analysisTargets,
+            bool computeTargetsHash,
+            IEnumerable<string> invocationInfoTokensToRedact)
+        {
+            var runInfo = new RunInfo();
+
+            if (analysisTargets != null)
+            {
+                runInfo.AnalysisTargets = new List<FileReference>();
+
+                foreach (string target in analysisTargets)
+                {
+                    var fileReference = new FileReference()
+                    {
+                        Uri = target.CreateUriForJsonSerialization(),
+                    };
+
+                    if (computeTargetsHash)
+                    {
+                        string sha256Hash = HashUtilities.ComputeSha256Hash(target) ?? "[could not compute file hash]";
```

Comment:
It seems to me that there are two options:

1. We care that we couldn't compute the file hash. Then either:
    1. Throw an exception, or
    2. Synthesize a result object to record the problem and add it to the Results list.
2. We don't care that we couldn't compute the file hash. Don't include the hash to the log file.

Replacing the hash with a message like this feels wrong to me. At some point a tool is going to come along, read the log file, compute the hash of the target, compare it to this string, see that it doesn't match, and complain.